### ISSUE TITLE
    Website: SoMe-Buttons einrichten:

### ISSUE THREAD

**Issue by OttoCFF** [StrategieLukas/stadt-land-klima#161]
LinkedIn – https://www.linkedin.com/company/stadt-land-klima
    Insta – https://www.instagram.com/stadt.land.klima/

**Comment by OttoCFF** [comment]
Ja, das ist umgesetzt.

Aber am iPhone ist sie Teile mit den SoMe-Buttons auf allen Seiten ausgeblendet.

**Comment by StrategieLukas** [comment]
Oh; habs jetzt gefixed und funktioniert auch für das mobile layout jetzt

**Comment by StrategieLukas** [comment]
ja, ist noch nicht live und auf staging gibt es gerade noch technische Probleme
Lokal sieht es aber schon so aus:

<img width="407" height="445" alt="Image" src="https://github.com/user-attachments/assets/c9655e64-c80f-4ecc-801c-2d6b8c94fefc" />

**Comment by OttoCFF** [comment]
Ich komme zurzeit nicht auf staging

**Comment by StrategieLukas** [comment]
Todo: Ergänzen um WhatsApp, Telegram und Signal für die Infokanäle
